packages/server/test/e2e/6_video_compression_spec.js

```diff
@@ -45,6 +45,21 @@ describe('e2e video compression', () => {
               expect(durationMs).to.be.closeTo(EXPECTED_DURATION_MS, humanInterval('15 seconds'))
             })
           })
+          .then((files) => {
+            return videoCapture.getChapters(files[0])
+            .then(({ chapters }) => {
+              // There are 40 chapters but we test only the first one
+              // because what we want to check is if chapters are added properly.
+              // In a chapter object, there are properties like 'end' and 'end_time'.
+              // We don't check them here because they return the test time in milliseconds.
+              // They cannot be guessed correctly and they can cause flakiness.
+              expect(chapters[0].id).to.eq(0)
+              expect(chapters[0].start).to.eq(0)
+              expect(chapters[0].start_time).to.eq(0)
+              expect(chapters[0]['TAG:title']).to.eq('num: 1 makes some long tests')
+              expect(chapters[0].time_base).to.eq('1/1000')
```

Comment:
do you think it's worth testing that the end_time is roughly `MS_PER_TEST` after the start_time?

Reply:
When I tested them with my local Ubuntu 20.04, the numbers were 3790 and 3157. And `MS_PER_TEST` is 500. I think it's hard to guess the number from `MS_PER_TEST`. And it's much bigger than this, because the setup time (loading and rendering page) is added.

I believe what we need is that `end` and `end_time` are number and not a `NaN`. That's why I added assertions like below: 

```js
expect(chapters[0].end).to.be.a('number')
expect(Number.isNaN(chapters[0].end)).to.be.false
expect(chapters[0].end_time).to.be.a('number')
expect(Number.isNaN(chapters[0].end_time)).to.be.false
```